test/python/WMCore_t/Services_t/ReqMgr_t/ReqMgr_t.py

```diff
@@ -140,7 +140,7 @@ def testRequestSimpleCycle(self):
         self.assertEqual(len(response), 1)
         clonedName = response[0]['request']
         response = self.reqSvc.getRequestByNames(clonedName)
-        self.assertEqual(response[0][clonedName]['TimePerEvent'], 15)
+        self.assertEqual(response[0][clonedName]['TimePerEvent'], 73.85)
```

Comment:
could you please put comment why this number should be 73.85?

Reply:
this value comes from the json template we load in the setUp() step